Sources/Sentry/SentryOptions.m

```diff
@@ -708,4 +709,136 @@ - (void)setEnablePreWarmedAppStartTracing:(BOOL)enablePreWarmedAppStartTracing
 
 #endif // SENTRY_UIKIT_AVAILABLE
 
+#if defined(DEGUG)
+- (NSString *)description
```

Comment:
`m`: I think adding a test for this would be great. First, it validates that the code doesn't crash, and it also validates that the description works correctly. It could cost a debugging developer plenty of time if it doesn't.

Reply:
I'm simply going to test that it returns non nil, so we know it doesn't crash. Testing for the actual value is going to be much harder, because several things could change that would result in a different string. I say let's cross that bridge if we get to it.